src/core/CoreStore.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.20;
+
+import {MsgInitiateTx, MsgInitiateTxResponse} from "../proto/cross/core/initiator/Initiator.sol";
+import {Account} from "../proto/cross/core/auth/Auth.sol";
+import {CoordinatorState} from "src/proto/cross/core/atomic/simple/AtomicSimple.sol";
+
+abstract contract CoreStore {
```

Comment:
I think `CrossStore` would be better. What was the reason for choosing `CoreStore`?